lib/simctl.js

```diff
@@ -32,7 +27,14 @@ async function launch (udid:string, bundleId:string):void {
 
 async function createDevice (name:string, deviceTypeId:string,
     runtimeId:string):void {
-  await simExec("create", 0, [name, deviceTypeId, runtimeId]);
+
+  let out;
+  try {
```

Comment:
Just wondering why "create" is the only command with try/catch/trim? Maybe this should be generic.